test/node/uuid_tests.js

```diff
@@ -162,4 +162,42 @@ describe('UUID', () => {
     const uuid = new UUID(UPPERCASE_DASH_SEPARATED_UUID_STRING);
     expect(inspect(uuid)).to.equal(`new UUID("${LOWERCASE_DASH_SEPARATED_UUID_STRING}")`);
   });
+
+  describe('deserialize', () => {
+    const serializedUUID = BSON.serialize({ uuid: new BSON.UUID().toBinary() });
```

Comment:
As was mentioned by @dariakp in our virtual office time, can we also here save the UUID itself to a variable and then assert the deserialised value is also the same in these tests and not just has the correct `_bsontype`?

Reply:
That sounds good!